modules/admin_manual/pages/configuration/files/manual_ file_ locking.adoc

```diff
@@ -0,0 +1,35 @@
+= Manual File Locking
+:toc: right
+
+== Introduction
+
+ownCloud Server 10.5 comes with great enhancements for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out).
```

Comment:
```suggestion
Starting with ownCloud Server 10.5 a new enhancement is provided for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out).
```